carbonmark/components/Activities/Activity.tsx

```diff
@@ -84,6 +86,26 @@ export const Activity = (props: Props) => {
       props.activity.amount && `${formatToTonnes(props.activity.amount)}t`;
   }
 
+  /** Expiration Labels */
+  if (isUpdateExpiration) {
+    if (
+      props.activity.timeStamp !== null &&
+      props.activity.timeStamp !== undefined
+    ) {
+      // The expiration is on the listing entity, but not activity. In order to avoid more extensive refactoring, 90 days is added the expiration here.
```

Comment:
Hmm we need to be careful here. 

Is the 90 days an unlinked external dependency? (i.e is 90 days hard coded somewhere else?)

Reply:
revised with DEFAULT_EXPIRATION_DAYS in place of the hardcoded 90